packages/subgraph/src/entities/StabilityDeposit.ts

```diff
@@ -8,37 +8,24 @@ import { beginChange, initChange, finishChange } from "./Change";
 import { getUser } from "./User";
 import { updateSystemStateByStabilityDepositChange } from "./SystemState";
 
-function getCurrentStabilityDepositOfOwner(_user: Address): StabilityDeposit | null {
-  let owner = getUser(_user);
+function getStabilityDeposit(_user: Address): StabilityDeposit {
+  let id = _user.toHexString();
+  let stabilityDepositOrNull = StabilityDeposit.load(id);
 
-  if (owner.currentStabilityDeposit == null) {
-    return null;
-  }
-
-  return StabilityDeposit.load(owner.currentStabilityDeposit);
-}
-
-function openNewStabilityDepositForOwner(_user: Address): StabilityDeposit {
-  let owner = getUser(_user);
-  let stabilityDepositSubId = owner.stabilityDepositCount++;
-
-  let newStabilityDeposit = new StabilityDeposit(
-    _user.toHexString() + "-" + stabilityDepositSubId.toString()
-  );
-  newStabilityDeposit.owner = owner.id;
-  newStabilityDeposit.depositedAmount = DECIMAL_ZERO;
-
-  owner.currentStabilityDeposit = newStabilityDeposit.id;
-  owner.save();
+  if (stabilityDepositOrNull != null) {
+    return stabilityDepositOrNull as StabilityDeposit;
+  } else {
+    let owner = getUser(_user);
+    let newStabilityDeposit = new StabilityDeposit(id);
 
-  return newStabilityDeposit;
-}
+    newStabilityDeposit.owner = owner.id;
```

Comment:
Is it more helpful to subgraph consumers to name this `.ownerId` rather than `owner`?